test/controllers/posts_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class PostsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
In a production app, you should most definitely add a set of specs for user management and posts management.
Do with it what you will.